### PR TITLE
#34 Redis provider

### PR DESCRIPTION
**Pull request by KamilWojciech**

### CODE REVIEW

**Comment by duzymaju** [comment]
:+1:

File: Resources/config/redis.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+services:
+    svd_scheduler.job_provider.redis:
+        arguments:
+            config:
+                scheme:                     tcp
+                host:                       127.0.0.1
+                port:                       6379
+        class:                          Svd\SchedulerBundle\Provider\Job\Redis
```
Comment (kedrap) [inline Resources/config/redis.yml]: Fix spacing

File: Resources/config/redis.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+services:
+    svd_scheduler.job_provider.redis:
+        arguments:
+            config:
+                scheme:                     tcp
```
Comment (kedrap) [inline Resources/config/redis.yml]: Fix order: 'scheme' to bottom

**Comment by kedrap** [comment]
:+1:

**Pull request merged**